ui/component/fileActions/view.jsx

```diff
@@ -120,15 +118,13 @@ export default function FileActions(props: Props) {
       <ClaimCollectionAddButton uri={uri} fileAction />
 
       {!hideRepost && !isMobile && !isLivestreamClaim && (
-        <Tooltip title={__('Repost')} arrow={false}>
+        <Tooltip title={__('Repost this claim')} arrow={false}>
```

Comment:
In general, we are avoiding the use of "claim" in Odysee to simplify it from lbry-desktop.
I think the original should be ok?